recipe/meta.yaml

```diff
@@ -27,19 +27,26 @@ requirements:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - mmh3
   requires:
     - pip
+    - pytest
```

Comment:
Does it not use `mypy` in the tests?  Just curious as it is listed as a test dependency.

Reply:
It doesn't. They run mypy in the [github build action](https://github.com/hajimes/mmh3/blob/ba23958677b4ba5153fd7d07f3f1d56cbdf56deb/.github/workflows/build.yml#L44-L45), but it's not in the tests.